src/components/Activity/TimeTag.js

```diff
@@ -1,30 +1,45 @@
-import React, { useEffect, useState } from 'react'
+import React from 'react'
 import PropTypes from 'prop-types'
-import { formatDistance } from 'date-fns'
+import { font, theme } from '@aragon/ui'
+import { formatDistance, formatDistanceStrict } from 'date-fns'
+import { useNow } from '../../hooks'
 
-function TimeTag({ style, date }) {
-  const [relativeTime, setRelativeTime] = useState('')
-
-  useEffect(() => {
-    const interval = setInterval(() => {
-      const relativeTime = formatDistance(new Date(), new Date(date))
-        .replace('about', '')
-        .replace('minutes', 'min')
-        .trim()
-      setRelativeTime(relativeTime)
-    }, 1000)
-
-    return () => {
-      clearInterval(interval)
-    }
-  }, [date])
+function getRelativeTime(now, targetDate) {
+  const past = targetDate < now
+  const fn = past ? formatDistance : formatDistanceStrict
+  const options = { addSuffix: true }
```

Comment:
Adding new properties to objects always makes me look, what do you think about this shorthand:
`const options = { addSufix: true, ...(past && { units: 'minute' } ) }`

Reply:
No strong opinion here, I added your suggestion thanks! :+1: